### DISCUSSION TITLE
GitHub éducation benefits not received yet after one week

### DISCUSSION THREAD

**Discussion by roombadra** [community/community#145295] [GitHub Education]
### Hi everyone,

<img width="369" alt="Capture d’écran 2024-11-22 à 11 31 54" src="https://github.com/user-attachments/assets/6c7ed12d-60dd-48cb-9bb1-87a0f5281ba3">

I have been approved for GitHub Education program on 16th November as you can see in screenshot provided above, and I have not received the benefits yet. And I have not received  any email regarding this matter. I already checked my email including spam folder.

**Comment by davevad93** [comment]
Hi @roombadra ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @roombadra was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by roombadra** [reply]
Hi @davevad93 thanks

**Reply by davevad93** [reply]
You're welcome @roombadra 👍

**Reply by alexCobat1212** [reply]
@roombadra you got benefits ??

**Reply by roombadra** [reply]
> @roombadra you got benefits ??

Yeah i got it
